tests/integration/loki-tester/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
File not needed, top-level license applies.

Reply:
Ack, my bad. Should have removed it after `charmcraft init`.